resources/assets/lib/user-group-badge.tsx

```diff
@@ -15,28 +16,38 @@ export default function UserGroupBadge({group, modifiers}: Props) {
     return null;
   }
 
-  const style = osu.groupColour(group);
-
-  let blockClass = classWithModifiers('user-group-badge', {
-    probationary: group.is_probationary,
-    [group.identifier]: true,
-  });
-  blockClass += classWithModifiers('user-group-badge', modifiers, true);
-
-  const playModes: JSX.Element[] = (group.playmodes ?? []).map((mode) => <i key={mode} className={`fal fa-extra-mode-${mode}`} />);
-
-  return (
-    <div
-      className={blockClass}
-      data-label={group.short_name}
-      style={style}
-      title={group.name}
-    >
-      {playModes.length > 0 &&
-        <div className={'user-group-badge__modes'}>
-          {playModes}
-        </div>
-      }
-    </div>
-  );
+  let children;
+  let title = group.name;
+
+  if (group.playmodes != null && group.playmodes.length > 0) {
+    children = (
+      <div className='user-group-badge__modes'>
+        {group.playmodes.map((playmode) => (
+          <i key={playmode} className={`fal fa-extra-mode-${playmode}`} />
+        ))}
+      </div>
+    );
+
+    const playmodeNames = group.playmodes
+      .map((playmode) => osu.trans(`beatmaps.mode.${playmode}`))
+      .join(', ');
+
+    title += ` (${playmodeNames})`;
+  }
+
+  const props = {
+    children,
+    'className': classWithModifiers('user-group-badge', {
```

Comment:
also we just changed the lint rule so some of these quotes aren't needed anymore >_>